src/ctim/schemas/sighting.cljc

```diff
@@ -17,15 +17,23 @@
       :refer-macros
       [def-entity-type def-eq def-map-type]])]))
 
-(def-map-type SightingTarget
+(def-map-type TargetCoordinates
   (concat
    (f/required-entries
     (f/entry :type v/Sensor)
     (f/entry :observables [c/Observable])
     (f/entry :observed_time c/ObservedTime))
    (f/optional-entries
-    (f/entry :os f/any-str)
-    (f/entry :properties_data_tables rel/DataTableReference)))
+    (f/entry :os f/any-str)))
+  :description "Describes a target device where a sighting came from.")
+
+(def-map-type SensorCoordinates
+  (concat
+   (f/required-entries
+    (f/entry :type v/Sensor)
+    (f/entry :observables [c/Observable]))
+   (f/optional-entries
+    (f/entry :os f/any-str)))
   :description "Describes a target device where a sighting came from.")
```

Comment:
This description should be updated.  We should also make it more appropriate for coordinates.  "Contains the identifying observables or a target|sensor..."